frontend/packages/ceph-storage-plugin/src/utils/common-ocs-install-el.tsx

```diff
@@ -109,7 +109,7 @@ export const VALIDATIONS = (type: keyof typeof ValidationType, t: TFunction): Va
         variant: AlertVariant.danger,
         title: t('ceph-storage-plugin~Public Network Attachment Definition cannot be empty'),
         text: t(
-          'ceph-storage-plugin~To use Multus networking the public Network Attachment Definition must be selected.',
+          'ceph-storage-plugin~To use Multus networking either the public or cluster Network Attachment Definition must be selected.',
```

Comment:
We need to change the title -  `Public Network Attachment Definition cannot be empty`.